test/Table_Tests/src/Database/Snowflake_Spec.enso

```diff
@@ -0,0 +1,661 @@
+from Standard.Base import all
```

Comment:
I see a lot of this is still calling `Postgres`.

Maybe we should not commit this file at all if it's not yet usable?

Alternatively, I'd at least add a note at the top of the file that it is still WIP and not being run with the main suite.